manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "adserver-graphql",
-  "version": "1.0.0",
+  "version": "0.0.2",
```

Comment:
I'm changin this here back to 0.0.2 so that, when I deploy a 0.1.0, the relevanC app will automatically get this version.

Then, I'll remove the 1.0.0 from existence.